db/schema.rb

```diff
@@ -10,14 +10,13 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_06_08_035549) do
+ActiveRecord::Schema.define(version: 2022_06_21_165259) do
 
   # These are extensions that must be enabled in order to support this database
-  enable_extension "pg_stat_statements"
```

Comment:
Can you manually adjust the schema file here so the only thing that is committed to this PR is the new index and the version date on line 13?